components/drops/useDrops.ts

```diff
@@ -1,51 +1,54 @@
 import { CollectionWithMeta } from '../rmrk/service/scheme'
+import { getDropById, getDrops } from '@/services/waifu'
 import unlockableCollectionById from '@/queries/subsquid/general/unlockableCollectionById.graphql'
 
 export interface Drop {
   collection: CollectionWithMeta
+  chain: string
   minted: number
   max: number
   dropStartTime: Date
-}
-
-interface DropsData {
-  drops: Drop[]
-  futureDrops: Drop[]
+  price: string
+  alias: string
 }
 
 const futureDate = new Date()
 futureDate.setDate(futureDate.getDate() * 7) // i weeks in the future
 
-export function useDrops(collectionId: string, clientName?: string) {
-  const dropsData = ref<DropsData>({
-    drops: [],
-    futureDrops: [],
-  })
+export function useDrops() {
+  const drops = ref<Drop[]>([])
+  onMounted(async () => {
```

Comment:
any other option than `onMounted`?

Reply:
Sorry but I don't know what's wrong with using `onMounted` ?

Reply:
nvm vue doc tell it's ok to use onMounted in composable